docs/how-to-guides/deploy-a-flow/deploy-using-docker.md

```diff
@@ -83,7 +83,38 @@ name: open_ai_connection
 module: promptflow.connections
 api_key: ${env:OPEN_AI_CONNECTION_API_KEY} # env reference
 ```
-You'll need to set up the environment variables in the container to make the connections work.
+You'll need to set up the environment variables in the container to make the connections work. For a full list of environment variables see [Load from environment variables](https://microsoft.github.io/promptflow/how-to-guides/manage-connections.html#load-from-environment-variables).
+
+#### Configuring the run and finish script
+
+Depending on your use case you made need to make changes to how your application runs. Out of the box, running `pf flow build...` will generate two scripts located under `runit/promptflow-serve/`, a `run` script used to initialise the flow, and a `finish` script used to gracefully shutdown the flow. These two scripts are where you can initialise and shutdown additional applications that run alongside your flow.
+
+For example, lets assume you want to add you need to run [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) within the same container.
```

Comment:
"lets assume you need to run".

Lets also mention that running OTEL collector inside the same container is only an example (or for scenarios where you can't run another container) but the preferred way is to run it outside.

Reply:
How about something like:

> For the purpose of this example, we will install and run [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) within the same container.  Please note that this is just an example for scenarios where you cannot run additional containers alongside your flow's container.

Reply:
Resolved in [fabe936](https://github.com/microsoft/promptflow/pull/3508/commits/fabe936f7f89c927c89cc06116295486b3923746).